### ISSUE TITLE
fix standalone model generator dependency

### ISSUE THREAD

**Issue by osrf-migration** [osrf/drcsim#420]
**Original report ([archived issue](https://osrf-migration.github.io/drcsim-gh-pages/#!/osrf/drcsim/issues/420)) by John Hsu (Bitbucket: [hsu](https://bitbucket.org/%7B0a186eae-abf0-4514-a951-23db5eccc286%7D/), GitHub: [hsu](https://github.com/hsu)).**

----------------------------------------
Changing `atlas_description/urdf/*` does not trigger standalone model generator to regenerate affected models.

**Comment by osrf-migration** [comment]
**Original comment by Nate Koenig (Bitbucket: [Nathan Koenig](https://bitbucket.org/%7Bc862cdd9-fcc8-4419-9fc7-e20db14b8fcb%7D/)).**

----------------------------------------

* set assignee_account_id to "557058:f3968cd3-4910-4384-8349-482a6c7889ec"
* set assignee to "hsu (Bitbucket: [hsu](https://bitbucket.org/hsu), GitHub: [hsu](https://github.com/hsu))"

**Comment by osrf-migration** [comment]
**Original comment by John Hsu (Bitbucket: [hsu](https://bitbucket.org/%7B0a186eae-abf0-4514-a951-23db5eccc286%7D/), GitHub: [hsu](https://github.com/hsu)).**

----------------------------------------
to pull request once bitbucket outage is fixed...
~~~
diff -r 0d6f86bff85b902717ff2cd804338b15e4fcb1b1 tools/URDF_helpers.cmake
--- a/tools/URDF_helpers.cmake	Tue Sep 16 20:13:50 2014 -0700
+++ b/tools/URDF_helpers.cmake	Tue Sep 16 20:14:06 2014 -0700
@@ -40,6 +40,5 @@
                          DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/robots/${xacro})
     endif()
     add_custom_target(${urdf}_gen ALL DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/${standalone_name}/${base}/${urdf})
-    add_dependencies(${urdf}_gen ${CMAKE_CURRENT_SOURCE_DIR}/robots/${xacro})
   endforeach()
 endmacro()
~~~

**Comment by osrf-migration** [comment]
**Original comment by John Hsu (Bitbucket: [hsu](https://bitbucket.org/%7B0a186eae-abf0-4514-a951-23db5eccc286%7D/), GitHub: [hsu](https://github.com/hsu)).**

----------------------------------------

* changed state from "new" to "resolved"

fix issue #420, make compile-time generated standalone model depend on xacro

→ \<\<cset 0d6f86bff85b902717ff2cd804338b15e4fcb1b1\>\>

**Comment by osrf-migration** [comment]
**Original comment by John Hsu (Bitbucket: [hsu](https://bitbucket.org/%7B0a186eae-abf0-4514-a951-23db5eccc286%7D/), GitHub: [hsu](https://github.com/hsu)).**

----------------------------------------
[pull request #461](https://osrf-migration.github.io/drcsim-gh-pages/#!/osrf/drcsim/pull-requests/461)

**Comment by osrf-migration** [comment]
**Original comment by John Hsu (Bitbucket: [hsu](https://bitbucket.org/%7B0a186eae-abf0-4514-a951-23db5eccc286%7D/), GitHub: [hsu](https://github.com/hsu)).**

----------------------------------------

* changed state from "resolved" to "open"

undo auto-close, [pull request #461](https://osrf-migration.github.io/drcsim-gh-pages/#!/osrf/drcsim/pull-requests/461) has a partial fix, but insufficient.

**Comment by osrf-migration** [comment]
**Original comment by John Hsu (Bitbucket: [hsu](https://bitbucket.org/%7B0a186eae-abf0-4514-a951-23db5eccc286%7D/), GitHub: [hsu](https://github.com/hsu)).**

----------------------------------------

* changed state from "open" to "resolved"

Merged in issue_420 ([pull request #461](https://osrf-migration.github.io/drcsim-gh-pages/#!/osrf/drcsim/pull-requests/461))

fix issue #420, make compile-time generated standalone model depend on xacro

→ \<\<cset 861e802c76d11a2c2d1798f44cb97008145b598a\>\>
